src/bin/rustfmt.rs

```diff
@@ -168,6 +181,11 @@ fn make_opts() -> Options {
                 "Recursively searches the given path for the rustfmt.toml config file. If not \
                  found reverts to the input file path",
                 "[Path for the configuration file]");
+    opts.optmulti("",
+                  "experimental-file-lines",
+                  "Format specified line RANGEs in FILE. RANGEs are inclusive of both endpoints. \
+                  May be specified multiple times.",
+                  "FILE:RANGE,RANGE,...");
```

Comment:
Can you document this more thoroughly somewhere please (maybe in the README), needs good explanations and a concrete example.